src/System/Taffybar/Pager.hs

```diff
@@ -102,6 +104,7 @@ defaultPagerConfig   = PagerConfig
   , fillEmptyImages  = False
   , preferCustomIcon = False
   , customIcon       = \_ _ -> Nothing
+  , imageCount       = 4
```

Comment:
for my taffybar, 4 icons completely fills my taffybar, with nothing but wmlog.
(my wmlog is ~20% of my bar already)